tests/phpunit/tests/rest-api/rest-revisions-controller.php

```diff
@@ -8,24 +8,29 @@
  * @group restapi
  */
 class WP_Test_REST_Revisions_Controller extends WP_Test_REST_Controller_Testcase {
-	protected static $post_id;
```

Comment:
In the interest of minimizing churn, can we keep all of the existing variables named as they are?

I recognize `post_id_1` is easier to follow, but all of the churn makes the history of the file much more difficult to follow.

Reply:
Fair enough. I think it's still worth removing some of the cognitive load of grokking these variables, so I'll revert the change here and create another patch.

Thanks!

Reply:
Thanks! Lot easier to read this way.